#157906013 Custom theme

chuzksy-codeactive opened the pull request:
#### What does this PR do?
Create a new reaction theme for shop

#### Description of Task to be completed?
Users should see a new theme when logged into the app and when they receive an email from the vendor
#### How should this be manually tested?
clone repo, install package dependencies by running: `meteor npm install` on the command prompt
Then run `reaction` or `reaction run`

#### Screenshots (if appropriate)
<img width="622" alt="screen shot 2018-06-04 at 5 25 00 am" src="https://user-images.githubusercontent.com/32555808/40898178-b479f0a0-67b7-11e8-8ed0-959680ee5415.png">
The mage above shows how each email template would look like. Note that the sample colours provided in the PT user story are applied here.

Commit:
feature(customize): create theme
- create a custom theme in navbar.less
- import the navbar.less file and register it
- create a inline script to customize the email templates
[Finishes #157906013]

Commit:
feature(customize): create theme
- create a custom theme in navbar.less
- import the navbar.less file and register it
- create a inline script to customize the email templates
[Finishes #157906013]

Commit:
feature(customize): create theme
- create a custom theme in navbar.less
- import the navbar.less file and register it
- create a inline script to customize the email templates
[Finishes #157906013]

emmabaye commented on imports/plugins/custom/thorinrc-theme/client/index.less:
```diff
@@ -0,0 +1 @@
+@import "styles/navbar.less";
```
There should be a new line at the end of this file.

chuzksy-codeactive replied:
@emmabaye Thank you for your review, for the deployment @amarachukwu-agbo  is working on it. And I have added a new line.

emmabaye reviewed (commented):
Nice UI. It would have been better if this PR was deployed.

Commit:
Pulled from staging to update app

Commit:
Pulled from staging to update app

Commit:
feature(customize): create theme
- create a custom theme in navbar.less
- import the navbar.less file and register it
- create a inline script to customize the email templates
[Fixes #157906013]

Orlayhemmy reviewed (commented):
Good Job
